deployment/docker-compose.yml

```diff
@@ -1,8 +1,26 @@
+# Production-ready setup.
+# Uses the newest `stable` image of the `master` branch.
+# a traefik instance handles exposition of service instances.
+# All services are expected to be started, hence the following command
+# must be run to start the docker setup:
+#
+#   docker-compose up
+#
+# Add the `-d` flag to daemonize the containers.
+#
+# Updating the images is as easy as this:
+#
+#   docker-compose pull && docker-compose restart
+#
+# ..Note::
+#
+#   The above commands assume you're already in the directory where
+#   where this docker-compose file is located.
 version: '3.4'
 
 x-defaults: &defaults
   restart: always
-  image: raiden-services
+  image: raidennetwork/raiden-services:stable
```

Comment:
how do we get these to build?

Reply:
The build config for these can be found here:
https://cloud.docker.com/u/raidennetwork/repository/docker/raidennetwork/raiden-services/builds/edit

So you'll need a tag to be pushed to master in order to have that build.

Reply:
there is a tag, `v0.1.0` exists and should be picked up afaict